extensions/interactions/SetInput/directives/SetInput.js

```diff
@@ -64,6 +64,15 @@ oppia.directive('oppiaInteractiveSetInput', [
             return false;
           };
 
+          var hasBlankOption = function(answer) {
+            for (var i = 0; i < answer.length; i++) {
```

Comment:
Take a look at Array.some() -- may be cleaner.

Reply:
Ok I'll take a look at that

Reply:
Done. Looks a lot cleaner now. Thanks